### ISSUE TITLE
VideoCapturerAndroid is deprecated in webrtc:1.0.22379

### ISSUE THREAD

**Issue by pavelsust** [vivek1794/webrtc-android-codelab#24]
Hello, 
I was following your code but in that time i figure out that VideoCapturerAndroid  is deprecated and in your step-2 you are using library from lib. 
in step-1 you show how to capture video ok i agree with you. But if i can't use VideoCapturerAndroid  then how i manage events or handle the CustomCameraEventsHandler which implements CameraEventsHandler.
If you figure out any way please share with us.

**Comment by wnieves19** [comment]
Try using:
implementation 'org.webrtc:google-webrtc:1.0.22672'
in your app gradle file.

**Comment by paveltech** [comment]
@wnieves19 is ICE server is running??

**Comment by wnieves19** [comment]
If you mean the getIceServers method in MainActivity.java, that API (xirsys) has changed, and you have now you have to change your retrofit call to be able to fetch the IceServers

**Comment by paveltech** [comment]
I think the API is not working now like invalid cause it's response is null.

**Comment by wnieves19** [comment]
You're correct it's not working because the API has changed, I have fix it in a fork, and submitted a Pull Request.

**Comment by paveltech** [comment]
i have used our own turn server.

**Comment by cannizarro** [comment]
@paveltech @wnieves19  Have you been able to find any work arounds for VideoCapturerAndroid.create()? Please help.

**Comment by Hassan99** [comment]
VideoCapturer capturer = VideoCapturerAndroid.create(name, eventsHandler);

here "VideoCapturerAndroid" not found in this library. could you please give solution for this?
